examples/jsm/renderers/webgpu/WebGPUUniformBuffer.js

```diff
@@ -20,9 +20,7 @@ class WebGPUUniformBuffer extends WebGPUBinding {
 
 	getByteLength() {
 
-		const buffer = 	this.buffer;
-
-		return buffer.byteLength + ( ( 4 - ( buffer.byteLength % 4 ) ) % 4 ); // ensure 4 byte alignment, see #20441
+		return WebGPUUniformBuffer.getByteLength(  this.buffer.byteLength );
```

Comment:
I'm not sure this change is right. When I remember correctly, there was a use case where 4 instead of 16 bytes has to be used 🤔 .

Reply:
The variation of vectors fit ony if used 16 bytes as chuck size. I did some testes for example:

The default stride is 4 bytes.

| GLSL  | Buffer size | Chunk size | Error |
| :------------ |:---------------:| -----:| -----:|
| vec2[2]      | 8      |  16   | No errors
| vec2[2]      | 8      |  8    |  binding 1 given 24 bytes, required 32 bytes
| vec2[2]      | 8      |   4  | binding 2 given 16 bytes, required 32 bytes

Reply:
I think that the name `getByteLength` does not represent exactly. I think that `getFloatLength` is more correct, I will fix.

Reply:
Okay, let's try it like that.

Reply:
I update the example, you can change the `GPUChunkSize` for test.